.travis.yml

```diff
@@ -1,45 +0,0 @@
-sudo: required
-language: ruby
-cache: bundler
-services:
-- docker
-rvm:
-- 2.5
-env:
-  global:
-    - REGISTRY_HOST=pcr-internal.puppet.net
-    - DOCKER_REGISTRY=pe-and-platform
-    - BOLT_SUDO_USER=true
-    - secure: Gk8LaACXYEVpv5LIWEMOuH3sJP4CzB2aSvE1BUcfDWkI+Hdgr2by3w/nGbKpyVD+v2H8r0zXyVrbCJ/qzx2gCRxqKJ2GKJEsrStT+8z3BXRRRzwkThIBVyWKk9b9bTt8AE0G94I3BE4gJyIPfbX5XxnKcg7nJZGOmubZpUPQX+2SXSfy9EbtY9iismwK7LGtWv6l90cK2eSLZGvdsSKPo7cylOldXfdYIyeBtvsIL1juBaiINX52Zgt371+nX53fDSYOKdIDLuhNqX3zpNOuIJ9DUj4E7IJA7+XhHy77zL98VjHtPo5H4fmKyZ2k+xbYqOydc5OPGguKequsnyDo5npktDrkbswnjWMXNDu+wImAd+IwHG2lTamsAnOGQ+E6g2oK0R5fUL26XJ3lBnTRrsLDnlrvqYqFxt3MCR+o5+DnTirSVQJfrRVsIKTucWHlYLTOUWkVDrLavJqIbWHytEbMf/BXUcovlQzSgfu5/Y1GkUJBnthtbiZfTImmBLcrqKDD4PnDmvC1v9Z5KR78MYu7lFTe5C4STj2aR6bwvqjiPKm6kYG5etOFEyRJ+CbqD2QsdF2N6Ww/RFWovqVqQIWuGdhumDUTdmQAiiPxl12M0+kIH6NugpBD3gt4RT0sni/T+booDw6b3Ts4WJ8FW1/LPWdy7gVo9yOCL4FhjOw=
-jobs:
-  include:
-    - env: CHECK=rubocop
-    - env: CHECK=unit
-    - env: CHECK=modules
-    - env: CHECK=docs
-    - env: CHECK=integration
-before_script:
-- eval `ssh-agent`
-- cat Gemfile.lock
-  # Add users to test sudo on localhost
-- echo 'travis:travis' | sudo chpasswd
-  # Undo travis sudoers config
-- sudo sh -c "echo 'Defaults authenticate' >> /etc/sudoers"
-- sudo sh -c "echo 'travis  ALL=(ALL) PASSWD:ALL' >> /etc/sudoers"
-script:
-- bundle exec rake travis:$CHECK
-after_script:
-- |
-  sync
-  export line_sep=$(printf '%*s' 80 | tr ' ' '#')
-  docker ps --all -q | xargs -L 1 -I % sh -c 'echo "\n${line_sep}\nContainer Logs - $(docker inspect % --format={{.Name}})\n${line_sep}\n"; docker logs %'
-  sleep 3s
-deploy:
-  provider: script
-  on:
-    repo: puppetlabs/bolt
-    # deploy the master branch or tag pushes
-    condition: $TRAVIS_BRANCH == "master" || -n $TRAVIS_TAG
```

Comment:
Let's make sure to keep pushing the bolt server container to pcr internal onmerge to master. We use it in orch integration tests